org.mfat.systemdpilot/org.mfat.systemdpilot.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.mfat.systemdpilot
+runtime: org.gnome.Platform
+runtime-version: '45'
```

Comment:
45 is the latest

Reply:
Can you please advise which gnome runtime version should be used for maximum compatibility?

Reply:
47 is the latest.

45 is EOL.